### PR TITLE
SISRP-13264 - Build "Academic Dates..." card in delegate user's default view ("Toolbox")

### PR DESCRIPTION
**Pull request by dfkaye**
https://jira.berkeley.edu/browse/SISRP-13264

requested refactorings per @christianvuerings

### CODE REVIEW

File: public/dummy/json/academic_dates.json [hunk 1]
```diff
@@ -0,0 +1,66 @@
+{
+  "feed": {
+    "academicDates": [
+      {
+        "date": "Dec 29-30",
```
Comment (christianvuerings) [inline public/dummy/json/academic_dates.json]: @dfkaye see other PR, we should be sending over a proper date object from the back-end.

File: src/assets/javascripts/angular/controllers/widgets/academicDatesController.js [hunk 2]
```diff
@@ -6,115 +6,21 @@ var angular = require('angular');
...
-   * TODO: replace hard-coded data feed with TBD academicDateCalendar service
-   * API data.
-   */
-  var data = {
-    data: {
-      feed: {
-        academicDates: academicDates
-      }
-    }
+  var getAcademicDates = function() {
```
Comment (christianvuerings) [inline src/assets/javascripts/angular/controllers/widgets/academicDatesController.js]: @dfkaye thanks!

File: src/assets/javascripts/angular/factories/academicDatesFactory.js [hunk 3]
```diff
@@ -0,0 +1,19 @@
+'use strict';
+
+var angular = require('angular');
+
+/**
+ * Academics Factory
+ */
+angular.module('calcentral.factories').factory('academicDatesFactory', function(apiService) {
+  // var url = '/api/my/academic_dates';
+  var url = '/dummy/json/academic_dates.json';
```
Comment (christianvuerings) [inline src/assets/javascripts/angular/factories/academicDatesFactory.js]: @dfkaye awesome, will be way easier once the back-end is ready.

File: src/assets/stylesheets/_toolbox.scss [hunk 4]
```diff
@@ -59,15 +59,15 @@
...
-      display: table;
       padding: 5px 0;
       &:first-of-type {
         padding-top: 20px;
       }
       .cc-toolbox-widget-academic-calendar-date {
-        display: table-cell;
         padding: 0 5px 0 0;
-        white-space: nowrap;
+        &::after {
```
Comment (christianvuerings) [inline src/assets/stylesheets/_toolbox.scss]: @dfkaye could we just put the `:` in the `html`?

**Comment by christianvuerings** [comment]
lgtm - couple of small comments

File: src/assets/stylesheets/_toolbox.scss (hunk 4 above)
Comment (dfkaye) [inline src/assets/stylesheets/_toolbox.scss]: ok

File: public/dummy/json/academic_dates.json (hunk 1 above)
Comment (dfkaye) [inline public/dummy/json/academic_dates.json]: @christianvuerings I'd rather wait until the service is built, then revisit the JS and template to parse the new feed correctly.  Seems a big time sink to spend on growing temporary json that would have to be updated manually if the service isn't ready next time the dates has to be updated...

Comment (christianvuerings) [inline public/dummy/json/academic_dates.json]: @dfkaye that's fine - we just need to make sure the back-end devs don't see this file as the contract. Instead, as mentioned before, we would prefer CalCentral date objects.

/cc @raydavis @paulkerschen @johncrossman

**Comment by christianvuerings** [comment]
@dfkaye really appreciate doing this extra cleanup, thanks!

**Pull request merged**